src/en/components/side-navigation/base.md

```diff
@@ -0,0 +1,26 @@
+---
+github: https://github.com/cds-snc/gcds-components/tree/main/src/components/gcds-side-navigation
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?type=design&node-id=5633-11428&mode=design&t=4ltBpy3FPMc9pXcL-0
+permalink: false
+tags: ['sidenavigationEN', 'header']
+---
+
+# Side navigation <br>`<gcds-side-nav>`
+
+_Also called: sidebar menu, lefhand nav, vertical navigation._
+
+A side navigation is a vertical list of page links on the left side of the screen.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Side navigation component preview" "pt-400 pb-200" %}
+<gcds-side-nav label="Side navigation component preview">
+  <gcds-nav-link href="#">Nav link 1</gcds-nav-link>
+  <gcds-nav-link href="#">Nav link 2</gcds-nav-link>
+  <gcds-nav-group menu-label="nav-group" open-trigger="Nav group">
+    <gcds-nav-link href="#">Nav link 3</gcds-nav-link>
+    <gcds-nav-link href="#">Nav link 4</gcds-nav-link>
+  </gcds-nav-group>
+</gcds-side-nav>
```

Comment:
Do you think it would make sense to add one more nav-link after the group to be more aligned with the anatomy image? Or potentially adding the same nav link + nav group structure as in the anatomy image, but not having the first nav group expanded?